aten/src/ATen/native/native_functions.yaml

```diff
@@ -4117,14 +4117,14 @@
 
 - func: multinomial.out(Tensor self, int num_samples, bool replacement=False, *, Generator? generator=None, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CPU: multinomial_out_cpu
-    CUDA: legacy::cuda::_th_multinomial_out
+    CPU: multinomial_out
```

Comment:
Why can't we remove the dispatch?

Reply:
I debug it today, and it looks like, if I remove dispatch section, codegen will generate code that will not unpack variables, so it will fail when call to _th_multinomial_out: Expected Tensor (not Variable) for argument #0 'result' in call to _th_multinomial_out

Reply:
Ugh that is weird. Can you leave a comment here? Also, @gchanan, do you have any suggestions for how to resolve this?

Reply:
@ifedan this isn't land-blocking but we should figure it out / clean this up eventually.